src/main.rs

```diff
@@ -212,9 +213,12 @@ fn run_target(target: String) -> io::Result<bool> {
         return Err(io::Error::new(io::ErrorKind::Other, "Failed to build fuzz target"))
     }
 
+    let _ = fs::create_dir("corpus");
+
     // can't use cargo run since we can't pass -L args to it
     let path = format!("target/debug/{}", target);
     let mut run_cmd = process::Command::new(path);
+    run_cmd.arg("corpus");
```

Comment:
Shouldn’t pass the argument if creating the directory failed for any reason other than the directory already existing.